Description: Update to system-deps 6
Last-Update: 2022-10-03 <YYYY-MM-DD, last update of the meta-information, optional>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: rust-gobject-sys-0.14.0/Cargo.toml
===================================================================
--- rust-gobject-sys-0.14.0.orig/Cargo.toml
+++ rust-gobject-sys-0.14.0/Cargo.toml
@@ -56,7 +56,7 @@ version = "1.0.0"
 [dev-dependencies.tempfile]
 version = "3"
 [build-dependencies.system-deps]
-version = "3"
+version = "6"
 
 [features]
 dox = []
